src/ControlsPanel.cpp

```
@@ -88,7 +91,11 @@ long RoundL(double v) { return std::lround(v); }
 // text on labels. (Native controls on macOS may ignore colour changes.)
 void ThemeWindow(wxWindow* w, const MayaraTheme& t) {
   w->SetBackgroundColour(t.panel_bg);
-  if (wxDynamicCast(w, wxStaticText)) w->SetForegroundColour(t.text);
+  // Anything that draws its own text needs the foreground set too, not just
+  // labels: a native wxTextCtrl keeps the system text colour (black), which on
+  // this panel's near-black background is invisible until you select it.
+  if (wxDynamicCast(w, wxStaticText) || wxDynamicCast(w, wxTextCtrl))
```

Clang (14.0.6)

[warning] statement should be inside braces

(readability-braces-around-statements)